### DISCUSSION TITLE
shadcn/ui installation issue

### DISCUSSION THREAD

**Discussion by bi3-fih** [shadcn-ui/ui#2756] [General]
Hello guys how are you doing all , Ihave an issue using shadcn , now am developing a marketplace using Nextjs and i get so far some progress designing and styling the back-end , but when I am about to add some drop down menus I want to use shadcn but when I install it its ruins everything I have build. anyone knows how to fix this kind of issue please .

**Comment by leinardjg** [comment]
> Hello guys how are you doing all , Ihave an issue using shadcn , now am developing a marketplace using Nextjs and i get so far some progress designing and styling the back-end , but when I am about to add some drop down menus I want to use shadcn but when I install it its ruins everything I have build. anyone knows how to fix this kind of issue please .

Any details you can provide?
